datumaro/components/annotation.py

```diff
@@ -813,3 +815,45 @@ class DepthAnnotation(_ImageAnnotation):
     """
 
     _type = AnnotationType.depth_annotation
+
+
+@attrs(slots=True, order=False)
+class Skeleton(Annotation):
+    """
+    Represents a skeleton.
+    """
+
+    _type = AnnotationType.skeleton
+
+    elements: List[Points] = field(default=None)
+
+    label: Optional[int] = field(
+        converter=attr.converters.optional(int), default=None, kw_only=True
+    )
```

Comment:
I agree that this issue exists. The issue also existed when `Points` represented `Skeletons`.

In this case, I think that validation is needed for `PointsCategories`, because the number of labels inside `PointsCategories` doesn't match the number of vertices in the joints.

What do you think about it?

Reply:
Yes, I agree that validation for `PointCategories` is needed for this case.

Reply:
I think I will do it in another PR later